tpu_inference/layers/vllm/quantization/unquantized.py

```diff
@@ -1,4 +1,4 @@
-from typing import Any, Callable, Optional, Union
+from typing import Any, Optional, Union
 
 import jax
 import jax.numpy as jnp
```

Comment:
no need to change the file compressed_tensors_moe.py?

Reply:
ohhh good point. I'm surprised that the ci still passed despite this? maybe there's a missing unit test? Anyway, i've added the change.

Reply:
I suggest merging this pr for now - because the upstream issue is completing blocking our CI - and figure out a fix for this later. does that sounds good?

Reply:
because of https://github.com/vllm-project/tpu-inference/pull/1065#issue-3610239500